packages/core.rb

```diff
@@ -102,9 +103,12 @@ class Core < Package
   depends_on 'bundled_gems'
   depends_on 'slang'
   depends_on 'sqlite'
+  depends_on 'tar' if ARCH == 'armv7l'
   depends_on 'uchardet'
   depends_on 'unzip'
   depends_on 'upx'
+  depends_on 'uutils_coreutils' if ARCH == 'armv7l'
+  depends_on 'which' if ARCH == 'armv7l'
```

Comment:
Why does arm require all these additional packages?

Reply:
They bypass existing `aarch64` binaries.

Reply:
Why would armv7l be using aarch64 binaries in the first place?

Reply:
The current user space on the arm Chromebooks is primarily aarch64 with a fallback armv7l glibc to support armv7l libraries.

The same thing happens on x86_64 Linux installs that also have a i686 glibc installed to support i686 binaries.